lib/modules/manager/devcontainer/readme.md

```diff
@@ -0,0 +1,6 @@
+Extracts all Docker images from the `image` and `features` properties in these files:
+
+- `.devcontainer.json`
+- `.devcontainer/devcontainer.json`
+
+The `devcontainer` manager does _not_ support `build.dockerFile` or `dockerComposeFile` values as these are covered by the [`dockerfile`](../dockerfile/readme.md) and [`docker-compose`](../docker-compose/readme.md) managers respectively.
```

Comment:
@HonkingGoose again broken docs. we need to be more careful. those need to point to index.md not readme.md

```
WARNING -  Doc file 'modules/manager/devcontainer/index.md' contains a relative link '../dockerfile/readme.md', but the target 'modules/manager/dockerfile/readme.md' is not found among documentation files.
WARNING -  Doc file 'modules/manager/devcontainer/index.md' contains a relative link '../docker-compose/readme.md', but the target 'modules/manager/docker-compose/readme.md' is not found among documentation files.
```

Reply:
@viceice you are right! I'm sorry for not catching this when reviewing the PR!